src/sentry/auth/staff.py

```diff
@@ -31,9 +32,10 @@
 COOKIE_HTTPONLY = getattr(settings, "STAFF_COOKIE_HTTPONLY", True)
 
 # the maximum time the session can stay alive
-MAX_STAFF_SESSION_AGE = timedelta(hours=4)
+MAX_AGE = timedelta(hours=4)
```

Comment:
Changing this b/c this is the standard constant name in some of the package cookie functions